hadoop-common-project/hadoop-common/src/test/java/org/apache/hadoop/fs/TestPath.java

```diff
@@ -404,6 +404,8 @@ public void testGetName() {
   
   @Test (timeout = 30000)
   public void testAvroReflect() throws Exception {
+    // Avro expects explicitely stated, trusted packages used for (de-)serialization
+    System.setProperty("org.apache.avro.SERIALIZABLE_PACKAGES", "org.apache.hadoop.fs.Path");
```

Comment:
make the property name a constant somewhere in hadoop-common tests and refer to it in both places

Reply:
the property is for package names as opposed to class names - should really have value like `""org.apache.hadoop.fs"`